asciidoctorj-cli/src/main/java/org/asciidoctor/cli/jruby/AsciidoctorInvoker.java

```diff
@@ -99,13 +93,6 @@ private String getAsciidoctorJVersion() {
         }
     }
 
-    private void setTimingsMode(Asciidoctor asciidoctor, AsciidoctorCliOptions asciidoctorCliOptions, Options options) {
```

Comment:
Timings seem to have been removed without an alternative?

Reply:
Timings are initiated by setting the option `:timings` to `true`:
https://github.com/asciidoctor/asciidoctorj/pull/1249/files#diff-cfe4d6156f2de76a22ff5129622aa620750395c8a229f9725d597ec4dcd1292bR328
```
asciidoctorj -r asciidoctor-diagram -R .. -B $(pwd) -D target '**/*.adoc' -v -w --timings
OpenJDK 64-Bit Server VM warning: Options -Xverify:none and -noverify were deprecated in JDK 13 and will likely be removed in a future release.
Input file: sub1/test.adoc
  Time to read and parse source: 0.04277
  Time to convert document: 0.00150
  Total time (read, parse and convert): 0.04427
Input file: sub1/test2.adoc
  Time to read and parse source: 0.00904
  Time to convert document: 0.00052
  Total time (read, parse and convert): 0.00956
```